src/vrep_plugin.cpp

```diff
@@ -782,6 +786,72 @@ static mc_control::MCGlobalController & gc()
   return *gc_ptr;
 }
 
+static void setup_datastore()
+{
+  auto & ctl = gc_ptr->controller();
+  auto add_datastore_entries = [](mc_control::MCController & ctl) {
+    auto & ds = ctl.datastore();
+    /**
+     * @brief Controllers can check whether they are running in CoppeliaSim if this flag is present.
+     */
+    ds.make<bool>("CoppeliaSim", true);
+    /**
+     * @brief SetObjectParent: Changes the parent of an object
+     *
+     * @param object Object name in CoppeliaSim
+     * @param parent Name of the new Parent object
+     * @param keepInPlace When true, the object pose follows the parent object
+     */
+    ds.make_call("CoppeliaSim::SetObjectParent",
+                 [](const std::string & object, const std::string & parent, bool keepInPlace) -> bool {
```

Comment:
Should we expose the `keepInPlace` parameter? This seems mostly related to assembly in CoppeliaSim, we don't use those and it's not clear to me what the `false` effect is when assembles are not used

Reply:
I don't mind not exposing it. I haven't really understood what the actually behaviour is when this is false ;)